codebuild/codebuild_build_container_docker_hub/codebuild_build_container_docker_hub.yml

```diff
@@ -8,12 +8,11 @@ phases:
     commands:
       - docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD
       - ACTIONS_REQUIRED="${CODEBUILD_SRC_DIR_actions_required}/actions_required.json"
-      - if [ -n "$CHECK_TRIGGER" ]; then
-          TRIGGER_BUILD=$(jq --arg ACTION_NAME "$ACTION_NAME" '.[] | select(.action ==$ACTION_NAME).required' $ACTIONS_REQUIRED); 
-        else 
-          echo "CHECK_TRIGGER not supplied, defaulting to true..";
-          TRIGGER_BUILD=true; 
-        fi
+      - if [[ -z "$CHECK_TRIGGER" || "$CHECK_TRIGGER" -eq 0 ]]; then
+         echo "CHECK_TRIGGER not supplied";
```

Comment:
Do we want to change how `CHECK_TRIGGER` works in this situation? Before, if `CHECK_TRIGGER` was not supplied, `TRIGGER_BUILD` would be set to `true`. If we rely on that behaviour, this change might break some builds